typescript/src/events/reactions/rawReactionAddNotify.ts

```diff
@@ -19,10 +18,11 @@ export class UserEvent extends Event {
 	private kTimerSweeper: NodeJS.Timer | null = null;
 
 	public async run(data: LLRCData, emoji: string) {
-		const [allowList, channel, twemojiEnabled, ignoreChannels, ignoreReactionAdd, ignoreAllEvents, t] = await data.guild.readSettings(
+		const key = GuildSettings.Channels.Logs.Reaction;
+		const [allowList, channelId, twemojiEnabled, ignoreChannels, ignoreReactionAdd, ignoreAllEvents, t] = await data.guild.readSettings(
```

Comment:
```suggestion
		const [allowList, logChannelId, twemojiEnabled, ignoreChannels, ignoreReactionAdd, ignoreAllEvents, t] = await data.guild.readSettings(
```

And rename references accordingly. I had to check what the heck `channelId` was, a simple other name makes it easier already.